index.html

```diff
@@ -11,21 +11,24 @@
       type="module"
     ></script>
   </head>
-  <body class="flex justify-center">
-    <div class="flex-grow max-w-2xl mt-10 px-4">
-      <header>
-        <h1 class="font-bold mb-10 text(center gray-600 xl)">
-          Koodikrapula.fi
-        </h1>
+
+  <body class="flex justify-center bg-gray-50">
+    <div class="flex-grow max-w-2xl px-4">
+      <header class="flex justify-center">
+        <a href="#" class="hover:bg-gray-100 hover:text-red-400 my-6">
```

Comment:
Nit (non-blocking): the hover classes could be combined like the text classes on the next line. But we can do this later.